<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: qlineargradient(spread:pad, x1:0, y1:0.545, x2:0.977, y2:0.539773, stop:0 rgba(65, 51, 83, 255), stop:1 rgba(34, 27, 44, 255));</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_pinstatus">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>490</y>
         <width>781</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 30, 97);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>170</y>
         <width>781</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 245, 237);</string>
       </property>
       <property name="text">
        <string>Please enter your details to sign in</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_appname">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>781</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 107, 241);</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;ELITE&lt;/strong&gt; ATM </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_2_accountno">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>274</y>
         <width>200</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 170, 255);
background-color:none;</string>
       </property>
       <property name="text">
        <string>ACCOUNT NO</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3_pin">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>340</y>
         <width>200</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 170, 255);
background-color:none;</string>
       </property>
       <property name="text">
        <string>ATM PIN</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_accountno">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>260</y>
         <width>350</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
border: 1px solid white;
border-radius:10px;
color:white;
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2_pin">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>330</y>
         <width>350</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
border: 1px solid white;
border-radius:10px;
color:white;
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_new">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>430</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
color: rgb(174, 181, 189);
border: 0px;
background-color: none;
}

QPushButton:hover
{
color: rgb(255, 170, 255);
}</string>
       </property>
       <property name="text">
        <string>Create New Account</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_enter">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>400</y>
         <width>240</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-color: rgb(255, 0, 255);
font: 10pt &quot;Segoe UI&quot;;
color:white;
border-radius: 20px;
}

QPushButton:hover
{
background-color: rgb(170, 0, 255);
}</string>
       </property>
       <property name="text">
        <string>ENTER</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
